lib/admin/WP_Auth0_Admin_Generic.php

```diff
@@ -258,4 +258,12 @@ protected function get_docs_link( $path, $text = '' ) {
 		$text = empty( $text ) ? __( 'here', 'wp-auth0' ) : sanitize_text_field( $text );
 		return sprintf( '<a href="https://auth0.com/docs/%s" target="_blank">%s</a>', $path, $text );
 	}
+
+	protected function sanitize_switch_val( $val ) {
```

Comment:
This will handle values from the UI (which save a positive as `"1"`) as well as JSON-imported. Need to handle a few (but not all) truth-y values.